google/oauth2/challenges.py

```diff
@@ -25,6 +25,7 @@
 
 
 REAUTH_ORIGIN = "https://accounts.google.com"
+SAML_CHALLENGE_MESSAGE = "Please run `gcloud auth login --update-adc` to complete reauthentication with SAML."
```

Comment:
Because only gcloud users will trigger reauth, so this exception message will only be shown to gcloud users. I believe we don't need the "--update-adc" because it is to overwrite the ADC. We only need to ask user to run 'gcloud auth login' to refresh gcloud's credential.

Reply:
Since user credentials is created from ADC json file. If we don't update the ADC json file reauth will be triggered again, so this becomes a loop to the user.

Reply:
This exception is raise when we go through the challenges. If the credential is created from ADC json, enable_reauth_refresh is False and we will not go through challenges. So only CLI will set enable_reauth_refresh to True and may see this exceptions.

Reply:
removed "--update-adc".